RELEASE-NOTES.md

```diff
@@ -1,43 +1,37 @@
-v2.8.2
-@patternfly-build patternfly-build released this 22 days ago · 24 commits to master since this release
-
-2.8.2 (2019-05-28)
+# 2.8.2 (2019-05-28)
 This is a milestone release for PatternFly 4 - RC2 - it includes the following:
-Components
-
-Alert:
-
-Adjusted warning alert webfont icon font-size (#1805)
-Added in-line modifier (#1775)
-Card:
-
-Added a wrapper for the Actions in the top right, so that content wraps around it - similar to the Popover component. Added a logo/img wrapper that is always left aligned. (#1745)
-Removed flex from header (#1817)
-Updated font sizes. Card body and footer text size were changed to be 14px by default and header was changed to16px. (#2103)
-Card Demo: Added demo (#1716)
-
-Charts: Added CSS Variables for Charts (#1846)
-
-Content: Changed margin bottom for small element (#1843)
-
-Form/login/wizard: Fixed undefined issues with IE11 script (#1871)
-
-Nav:
-
-Fixed horizontal nav spacing, background color (#1798)
-Updated aria-label in examples (#1783)
-Pagination: Updated options menu aria-label (#1782)
-
-Popover: Scoped title class to popover component (#1857)
-
-Table: remove min-height from buttons in expansion toggle (#1818)
-
-Wizard:
-
-Fixed wizard closing tag (#1803)
-Set pf-c-wizard__main to grow and fill available height (#1781)
-Other
 
-Docs (global): Updated docs and examples to use sentence case (#1796)
-Guidelines: Added punctuation, formatting (#1810)
-Gutter: Updated mobile gutter spacing in layouts that have gutters (#1829)
+## Components
+### Alert
+- Adjusted warning alert webfont icon font-size (#1805)
+- Added in-line modifier (#1775)
+### Card
+- Added a wrapper for the Actions in the top right, so that content wraps around it - similar to the Popover component. Added a logo/img wrapper that is always left aligned. (#1745)
+- Removed flex from header (#1817)
+- Updated font sizes. Card body and footer text size were changed to be 14px by default and header was changed to16px. (#2103)
+- Card Demo: Added demo (#1716)
+### Charts
```

Comment:
Since charts aren't a component, we're just adding vars for the react charts package to consume, should it be listed here or under "Other"?

Reply:
Moved.